### PR TITLE
Feat-139: Add Tests for Table Component

### LINKED ISSUE #139: Create tests for the table element
In the previous sprint, we moved the table UI element to the UI kit for reuse.   This ticket allows us to create tests for the new element.

### PR DESCRIPTION
**Pull request by DorianKwan**
Closes #139 

### 🔥 Summary
This PR adds tests for the base table component.

### 😤 Problem / Goals
- There are currently no tests for the base table component
- We are repeating the same styled components for every table in BE

### 🤓 Solution
- Adds basic tests to base table component
- Adds and exports styled components that are being repeated throughout the BE frontend

### 🗒️ Additional Notes
- No Pagination has been tested in this PR as that component has not been migrated over to the UI Kit yet.  I will be recommending our new PM to create a ticket for that.

### CODE REVIEW

File: src/components/Tables/Table/partials/Table.styled.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  align-items: center;
+  flex-wrap: wrap;
+`;
+
+export const TotalRows = styled.p`
+  margin-right: 1.5rem;
+  white-space: nowrap;
+`;
+
+export const TableFooter = styled.div``;
```
Comment (valfaro123) [inline src/components/Tables/Table/partials/Table.styled.ts]: Is this still being used?

**Review by valfaro123 — APPROVED** [review]

File: src/components/Tables/Table/partials/Table.styled.ts (hunk 1 above)
Comment (DorianKwan) [reply src/components/Tables/Table/partials/Table.styled.ts]: Oh thanks, I forgot to add the styling there!

**Review by Thistle-Strum — APPROVED** [review]

File: src/mocks/mock-block.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  body: {
+    proposer:
+      '01c867ff3cf1d4e4e68fc00922fdcb740304def196e223091dee62012f444b9eba',
+    deploy_hashes: [],
+    transfer_hashes: [],
+  },
+  proofs: [],
+};
+
+export const createMockBlocks = (numberOfBlocks: number = 10) => {
```
Comment (mgibson-casper) [inline src/mocks/mock-block.ts]: If I recall we don't actually need to assign the type here if we're setting a default value.

**Review by mgibson-casper — APPROVED** [review]

File: src/mocks/mock-block.ts (hunk 2 above)
Comment (DorianKwan) [reply src/mocks/mock-block.ts]: Yeah it can infer -- I'll remove the type

**Pull request merged**